### PR TITLE
Fix #14

### LINKED ISSUE #14: tap interface names are truncated.
The openvpn config files we're generating have a significant prefix in the names for the tap devices that is the same:

https://github.com/zenhack/hil-vpn/blob/dba0ec73aa385451c576e95cb84a46ecbed08251/cmd/hil-vpn-privop/openvpn-conf.go#L16

The daemon adds some prefixing of its own, and apparently there is a relatively small cap on the length of interface names on linux. The upshot of all this is that all of the configs end up creating an interface called `tap-hil-vpn-hil`, so we can't actually start more than one vpn at a time without getting an in-use error.

We need to re-work the way we generate device names to avoid this.

### PR DESCRIPTION
**Pull request by zenhack**
Built on top of the other prs.

### CODE REVIEW

**Review by ianballou — APPROVED** [review]
Looks good

File: cmd/hil-vpn-privop/openvpn-conf.go [hunk 1]
```diff
@@ -80,6 +82,28 @@ func (cfg OpenVpnCfg) Save() error {
 	return err
 }
 
+// Return a cryptographically-random 12-character base64(url) encoded string.
+// This is to do collision avoidance given the 15-character limit on network
+// interface names. We still prefix interface names with tap for a modicum
+// of readability. See also issue #14.
```
Comment (naved001) [inline cmd/hil-vpn-privop/openvpn-conf.go]: We actually *need* to prefix interface names with `tap` if we don't explicitly specify `dev-type tap` in our openvpn configuration file (which we aren't). 
https://openvpn.net/community-resources/reference-manual-for-openvpn-2-4/

**Review by naved001 — COMMENTED** [review]
Just pointed out a detail, but otherwise looks good to me.

File: cmd/hil-vpn-privop/openvpn-conf.go (hunk 1 above)
Comment (zenhack) [reply cmd/hil-vpn-privop/openvpn-conf.go]: Good catch. I updated the comment to mention this.

**Review by naved001 — APPROVED** [review]

**Pull request merged**